### ISSUE TITLE
At the new dropdown can I add existing data source.

### ISSUE THREAD

**Issue by masud79** [Doptor-zz/Doptor#5]
At the new dropdown can I add existing data source. Means I have client module in list, in the invoice form/module how I can bring the client list at dropdown.

**Comment by dakshhmehta** [comment]
Right now, I don't think its possible natively, but once you create a module, you can easily do it by little modification to code.

Use following:

``` php
Form::select('client_id', Client::all()->lists('name', 'id'), Input::old('client_id'));
```

Hope this helps

**Comment by Doptor-zz** [comment]
Issue has been solved #30
